i18n/en.json

```diff
@@ -25,29 +25,29 @@
+	"requestwiki-error-invalidcomment": "Your wiki request contains an <strong>invalid</strong> character. Please try again, and if the problem persists [[Help center|please contact us]]",
+	"requestwiki-error-nodomain": "You must input a subdomain or custom domain.",
 	"requestwiki-error-notallfilledin": "You have to fill in at least the wiki name and language fields.",
-	"requestwiki-error-notime": "There has been a problem proccessing your request. Please submit your request again",
+	"requestwiki-error-notime": "There has been a problem proccessing your request. Please try again",
 	"requestwiki-error-patient": "You have already requested a wiki. Please be patient and one of our wiki creators should create it soon. If you need to request another wiki, please wait and you will be able to request later.",
 	"requestwiki-error-twodomains": "You specified both a subdomain and custom domain: you can only choose between one of them.",
-	"requestwiki-header": "You are going to request a new wiki at Miraheze, great! We need you to fill in this form, before we can review your request.\nYour wiki request will be reviewed by a volunteer, and it can be tracked at the page that will be given when submitting your request.\n\nIf you want a custom domain for your wiki (i.e. not a standard .miraheze.org subdomain), please read <span class=\"plainlinks\">[$1 this]</span> first.\n\nWhen you request a wiki, you automatically agree to our [[Dormancy Policy]].",
-	"requestwiki-label-comments": "Comments (e.g. what is your wiki about?):",
+	"requestwiki-header": "You are going to request a new wiki at Miraheze, great! We need you to fill in this form, before we can create your wiki,\nyour wiki request will be reviewed by a volunteer, and it can be tracked at the page that will be given when submitting your request.\n\nIf you want a custom domain for your wiki (i.e. not a standard .miraheze.org subdomain), please read <span class=\"plainlinks\">[$1 this]</span> first.\n\nWhen you request a wiki, you automatically agree to our [[Dormancy Policy]].",
+	"requestwiki-label-comments": "Reason (e.g. what is your wiki about?):",
 	"requestwiki-label-customdomain": "Custom domain:",
 	"requestwiki-label-language": "Language:",
 	"requestwiki-label-private": "Private wiki (not readable by everyone):",
 	"requestwiki-label-sitename": "Wiki name:",
 	"requestwiki-label-siteurl": "Subdomain:",
-	"requestwiki-notloggedin": "Sorry, you need to <span class=\"plainlinks\">[$1 log in]</span> before you can request a wiki.",
+	"requestwiki-notloggedin": "Sorry, you must <span class=\"plainlinks\">[$1 log in]</span> before you can request a wiki.",
 	"requestwiki-submit": "Submit",
 	"requestwiki-success": "Your wiki request ($1) was successfully submitted. Your request will be reviewed by a volunteer as soon as possible.",
-	"requestwikiqueue": "Wiki requests queue",
+	"requestwikiqueue": "Wiki requests",
```

Comment:
This is the queue for wiki requests, so we should probably keep the word "queue" in it to avoid any confusion.